### PR TITLE
Ignore sign key files when committing

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: android/.gitignore [hunk 1]
```diff
@@ -6,6 +6,8 @@ gradle-wrapper.jar
 /local.properties
 GeneratedPluginRegistrant.java
 build/
+*.jks
+*.ketstore
```
Comment (robertodoering) [inline android/.gitignore:10]:
```suggestion
*.keystore
```

**Review by robertodoering — COMMENTED** [review]
This isn't really necessary since I am not storing the key in the android folder in the first place.
Thanks for your contributions but what is your objective with this?

**Comment by Goooler** [comment]
> This isn't really necessary since I am not storing the key in the android folder in the first place.
> Thanks for your contributions but what is your objective with this?

Cause developer may copy a key from another project, and need to avoid to commit it every time.

**Comment by robertodoering** [comment]
> Cause developer may copy a key from another project, and need to avoid to commit it every time.

Sure but why would someone want to sign the app in the first place? If someone wants to fork harpy with the intention of contributing to the main project, it's totally fine to not sign the app when developing.

**Pull request closed without merging**